src/lectures.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Palestras do IMEsec
+---
+
+<div class="lectures">
+  <div class="lectures--header">
+    <a href="/">Voltar à página inicial</a>
+  </div>
+
+  <h1 class="heading lectures--title">Palestras</h1>
+  <p class="lectures--description">
+    Veja aqui as apresentações/materiais usados em nossas palestras:
+  </p>
+
+  <div class="lectures--items">
+    {% include lectures-item.html title="Seminário de Assembly" 
+    filename="seminario-assembly" description="" %} 
+
+    {% include lectures-item.html title="Stack Smashing" 
+    filename="stack-smashing" description="" %} 
+      
+    {% include lectures-item.html title="Phishing" 
+    filename="phishing" description="" %} 
+      
+    {% include lectures-item.html title="Workshop de Blockchain" 
+    filename="blockchain" description="" %} 
+      
+    {% include lectures-item.html title="Esteganografia 2" 
+    filename="esteganografia-2" description="" %} 
+      
+    
+    {% include lectures-item.html title="Wi-Fi - Parte 2" 
+    filename="wifi-parte-2" description="Descubra o que fazer quando se tem acesso ao roteador." %} 
+    
+    {% include lectures-item.html title="Wi-Fi - Parte 1"
+    filename="wifi-parte-1" description="" %} 
```

Comment:
swap these around. Makes more sense for the part 2 to be before part 1. Same for the stegano ones

Reply:
Hm, I talked about this with @r0zbot, but I forgot to mention it here. They seem to be in the "wrong order" because they are sorted in the release date descending order. Then, the newest presentation (that is the most likely a person will be looking for) will be the first one.

Is this order confusing?